tests/common.go

```diff
@@ -9199,6 +9191,39 @@ func GetClusterProvisionStatus() ([]ProvisionStatus, error) {
 			}
 		}
 	}
+	log.Infof("Cluster provision status [%v]", clusterProvision)
+	return clusterProvision, nil
+}
+
+func GetClusterProvisionStatus() ([]ProvisionStatus, error) {
+	// Using Node which is up and running
+	var selectedNode []node.Node
+	for _, eachNode := range node.GetNodes() {
```

Comment:
I have asked @vprabhakar-px to make some changes to this func, please check with him and rebase your changes on that

Reply:
yes